### PR TITLE
Project mongo api - Andrea

### PR DESCRIPTION
**Pull request by Andrealarsson**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,39 +1,113 @@
...
+  if (title) {
+    const bookInfo = await Book.find({
+      title: {
+        $regex: new RegExp(title, 'i')
+      }    
+    })
+    res.json(bookInfo)
+  }
+
+  // eslint-disable-next-line camelcase
```
Comment (darcywinterhill) [inline server.js:70]: When I first tried to enter a different language code than 'eng' I received all books instead. Then I noticed that your param is called language_code and in all other places (comments, README and in the '/' endpoint) you have written it as languages_code (and that's why nothing happened when I typed anything)  🐱‍👤

File: server.js [hunk 2]
```diff
@@ -1,39 +1,113 @@
...
+const app = express() 
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Endpoints: /books, /books?title=titanic, /books?languages_code=eng, /books/isbn/076790818X, /books/id/609bcffa120fc13f88868db7')
```
Comment (darcywinterhill) [inline server.js:49]: Good idea to have the endpoints here! ✨

File: server.js [hunk 3]
```diff
@@ -1,39 +1,113 @@
...
+// http://localhost:8080/books?title=titanic
+// http://localhost:8080/books?languages_code=eng
+app.get('/books', async (req, res) => {
+  // eslint-disable-next-line camelcase
+  const { title, language_code } = req.query
+
+  if (title) {
+    const bookInfo = await Book.find({
+      title: {
+        $regex: new RegExp(title, 'i')
```
Comment (darcywinterhill) [inline server.js:64]: Regex and 'i' 👌🏼

**Review by darcywinterhill — APPROVED** [review]
Clear code with relevant endpoints! ✨

File: server.js [hunk 4]
```diff
@@ -1,39 +1,113 @@
...
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const bookSchema = new mongoose.Schema({
+  bookID: Number,
```
Comment (KarolCislowski) [inline server.js:17]: You do not need to pass this id, because you will be provided with a mongo autogenerated _id

File: server.js [hunk 5]
```diff
@@ -1,39 +1,113 @@
...
+  // eslint-disable-next-line camelcase
+  const { title, language_code } = req.query
+
+  if (title) {
+    const bookInfo = await Book.find({
+      title: {
+        $regex: new RegExp(title, 'i')
+      }    
+    })
+    res.json(bookInfo)
```
Comment (KarolCislowski) [inline server.js:67]: nice filtering

**Review by KarolCislowski — APPROVED** [review]
Great API 🥇

**Pull request closed without merging**